jsconfig.json

```diff
@@ -0,0 +1 @@
+{"compilerOptions":{"target":"es6","experimentalDecorators":true},"exclude":["node_modules","bower_components","tmp","vendor",".git","dist"]}
```

Comment:
it's one of the vs-code default "decorator" files as mentioned above ... if we're not going to have these in the addon then we should add them to the `.gitignore` ok?